packages/at_secondary_server/lib/src/verb/handler/pkam_verb_handler.dart

```diff
@@ -131,17 +195,11 @@ class PkamVerbHandler extends AbstractVerbHandler {
       logger.finer('Exception in pkam signature verification: ${e.toString()}');
     }
     logger.finer('pkam auth:$isValidSignature');
-    // authenticate if signature is valid
-    if (isValidSignature) {
-      atConnectionMetadata.isAuthenticated = true;
-      atConnectionMetadata.authType = pkamAuthType;
-      atConnectionMetadata.enrollmentId = enrollId;
-      response.data = 'success';
-    } else {
-      atConnectionMetadata.isAuthenticated = false;
-      response.data = 'failure';
-      logger.severe('pkam authentication failed');
-      throw UnAuthenticatedException('pkam authentication failed');
-    }
+    return isValidSignature;
   }
 }
+
+class ApkamVerificationResult {
```

Comment:
This class creates a new Response object which is not necessary. You can use the response object in processVerb method to set the error code and description. 
The discussion yesterday was not to over load the response object to carry the APKAM public key. 
1. Remove this class.
2. Pass the response object in processVerb method only to set the error codes and description.
3. If the enrollment is approved, Use a local String variable to return the APKAM public key or just `return enrollDataStoreValue.apkamPublicKey`